lib/features/first_use/pages/welcome_page.dart

```diff
@@ -287,17 +288,21 @@ class _WelcomePageViewState extends State<WelcomePageView> {
         ),
       );
       await widget.prefs.remove('countryIso');
-      await get_it.initAPIService();
-      get_it.initRepositories();
```

Comment:
Are the other repositories initialised if you removed the methods?

Reply:
InitAPIService is unnecessary. But initRepositories unsure. I tried a solution of de-registering and then registering singletons like:
```
await getIt.unregister<ExampleRepo>();
getIt.registerLavySingleton<ExampleRepo>(()=>ExampleRepo());
```
But that solution was still doing double api calls. In the end in testing this current thing works. 

But in short: are they re-initialised? - no.